peps/pep-0735.rst

```diff
@@ -1052,7 +1066,7 @@ without relying on packaging metadata, and without trying to express their
 needs in packaging terms.
 
 Libraries
-'''''''''
+---------
 
 Libraries are python packages which build distributions (sdist and wheel) and
```

Comment:
```suggestion
Libraries are Python packages which build distributions (sdist and wheel) and
```

Also a few further up:

```
1037:For example, a source code repository may define python packaging metadata as
1040:The python application is built by copying the entire repository into a
```

And another [typo](https://github.com/python/peps/actions/runs/7598875290/job/20695312828?pr=3627#step:5:107):

```
peps/pep-0735.rst:1112: utlities ==> utilities
```

Reply:
Sorry I've been so slow getting back to this and applying the changes. I'll get these all done in a co-authored-by commit, which is why I didn't want to use the GitHub GUI for it.

I'm curious, do you have a linter (e.g. aspell, codespell) which is helping to catch these? I pretty often write `python` where I mean `Python`.

Reply:
I don't know if it's what Hugo is running, but you can run `make spellcheck` in the peps repo to get a report about typos.

Reply:
Yes, that's the one for the "utilities" typo, I looked at its (non-failing) output in the CI:

https://github.com/python/peps/pull/3627/checks

The "python" was by my own eye.

Reply:
Okay, thanks for the info; very useful!

I wonder if I can write myself a lint for `(^|\s)python` outside of specific contexts like `code-block`s... But good to know that I was missing an available linting tool already in the repo.